contrib/bin/makeHelm

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker build -t helm --build-arg HELM_VERSION=${HELM_VERSION} - <<EOF
+FROM ubuntu
+ARG HELM_VERSION=v2.1.3
+RUN apt-get -y update
+RUN apt-get -y upgrade
+RUN apt-get install -y curl
+RUN curl -L -o helm.gz http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz && \
+	tar -xf helm.gz  --strip-components=1 linux-amd64/helm && \
+	rm helm.gz
+WORKDIR /root/service-catalog
+ENTRYPOINT [ "/helm" ]
+EOF
```

Comment:
I'm ok with using an existing image, if one exists.  But if you mean pushing our own...do we need a new account on some registry? or are we allowed to just use Dockerhub and a personal namespace?